sklearn/feature_extraction/text.py

```diff
@@ -643,6 +643,24 @@ class CountVectorizer(BaseEstimator, VectorizerMixin):
 
         This is only available if no vocabulary was given.
 
+    Examples
+    --------
+    >>> from sklearn.feature_extraction.text import CountVectorizer
+    >>> #create document
+    >>> doc_test = ['Every summer our family gathers on the beach.',
+    ...             'We gather beach chairs on the beach.',
+    ...             'Our family gathers on the beach.',
+    ...             'Summer after summer we play the same games.']
+    >>> vectorizer = CountVectorizer(min_df=2, stop_words='english')
+    >>> X = vectorizer.fit_transform(doc_test)
+    >>> X.toarray()
+    array([[1, 1, 1, 1],
+       [2, 0, 0, 0],
+       [1, 1, 1, 0],
+       [0, 0, 0, 2]])
+    >>> print(vectorizer.get_feature_names())
```

Comment:
print one on each line so that you don't need to worry about discrepancies between python3 and python2 in representation of unicode strings